doc/architecture/sensor_data.adoc

```diff
@@ -1,9 +1,6 @@
 = Sensor data
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Et malesuada fames ac turpis egestas.
-Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
-Nulla facilisi morbi tempus iaculis urna id. Justo eget magna fermentum iaculis.
-Sed augue lacus viverra vitae congue eu. Velit ut tortor pretium viverra suspendisse.
-Amet commodo nulla facilisi nullam vehicula ipsum a. Nibh nisl condimentum id venenatis a.
-Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
\ No newline at end of file
+Sensor data imitates the output of real sensors.
+It is generated from ground-truth data, feature data or from sensor data.
```

Comment:
```suggestion
It can be generated from ground-truth data, sensor view data, feature data or from sensor data.
```

Reply:
And here also mention how it can be generated from sensor data? I guess you are referring to the fact that a logical model, for example a fusion model can take in sensor data and generate sensor data as well or?